src/legacy/core_plugins/data/public/filter/filter_service.ts

```diff
@@ -26,11 +27,12 @@ import { FilterManager } from './filter_manager';
 
 export interface FilterServiceDependencies {
   indexPatterns: IndexPatterns;
+  uiSettings: UiSettingsClientContract;
 }
 
 export class FilterService {
-  public setup({ indexPatterns }: FilterServiceDependencies) {
-    const filterManager = new FilterManager(indexPatterns);
+  public setup({ indexPatterns, uiSettings }: FilterServiceDependencies) {
+    const filterManager = new FilterManager(indexPatterns, uiSettings);
```

Comment:
nit: let's use one styling for similar files in one PR. See index pattern service. This logic placed inside the 'return' method.

![image](https://user-images.githubusercontent.com/20072247/62462791-f4b22500-b790-11e9-8342-858d0c60fd28.png)